### PR TITLE
feat: Add Uploading Multiple Files API

### LINKED ISSUE #20: [Feature] Multiple Files Support
### Current behavior:
`upload` takes only a single file at a time.
https://github.com/abramenal/cypress-file-upload/blob/master/README.md#api

### Desired behavior:
`upload` can take multiple files at a time.

### Versions
Chrome Version 72.0.3626.121

    "cypress-file-upload": "2.0.0",
    "cypress": "3.1.5",

### PR DESCRIPTION
**Pull request by kmui2**
Fixes #20 

Because uploading files will conflict with the current `upload` API, I created a new command called `uploadFiles`.

### CODE REVIEW

**Comment by abramenal** [comment]
What I am thinking now about is:
- create a single command `cy.uploadFiles` and force everyone to use files array even in case they have one (probably the worst one);
- create 2 commands: `cy.uploadFile` and `cy.uploadFiles` to easily distinguish between 'em;
- create a single command like `cy.upload` that can handle both single and multiple files (decide that internally via `Array.isArray` or anything else).. likely it's the **friendliest one for the user**, but the command name is not so descriptive.. Anyway in web dev world I think if you talk about upload, that means you upload a file but nothing else

**Comment by abramenal** [comment]
Anyway I think it's good to decide this now and make the approach generic so that after v3 we can just extend existing public API with various config sets rather than having another round of breaking changes.

@kmui2 @allout58 @virtuoushub @oliverodaa what do you think?

**Comment by kmui2** [comment]
I would go with 

> create a single command like cy.upload that can handle both single and multiple files

File: README.md [hunk 1]
```diff
@@ -47,18 +47,18 @@ Please see full version in [example](./example).
...
+cySubject.upload(fileOrArrayOfFiles, processingOptions);
 ```
 
-`file` is an object that represents file information and contains following properties:
+`fileOrArrayOfFiles` is an object (or an array of those) that represents file information and contains following properties:
 
-- (string) `fileContent` – raw file content
-- (string) `fileName` – file name with extension
-- (string) `mimeType` – file mime type (string)
+- {String} `fileContent` – raw file content
```
Comment (kmui2) [inline README.md]: Why did `String` become capitalized?

Comment (abramenal) [reply README.md]: It looks more like jsdoc now.
As for me it's a bit more fancy to have it capitalized, and officially it does not matter:
https://github.com/jsdoc3/jsdoc/issues/1046

**Comment by abramenal** [comment]
@allcontributors[bot] add kmui2 for code

**Comment by abramenal** [comment]
@kmui2 Thanks for your great effort, I'm glad adding you to contributors list! Keep on rocking!

**Pull request merged**